### DISCUSSION TITLE
P1 Style in Hammer

### DISCUSSION THREAD

**Discussion by CN1o2** [BEEmod/BEE2-items#4119] [General]
How can I make a P1 style map in Hammer?

I wish I could make a custom entrace but I don't know how to get the sounds and models implemented.

**Comment by slambo311** [comment]
All you really gotta do is apply the P1 style in BEE2 and then while making the map, just search through all the folders, and eventually, you'll find the models and stuff. The `bee2` and `props_map_editor` folders are the most likely to have them.
Also, this kind of stuff would've been answered much earlier if you said it on the BEE2 discord.
